pages/api/subscribe.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export default async function handler(req, res) {
+  const email = req.body.body;
+
+  try {
+    const user = await createUser(email);
+    res.status(200).json({ status: 'OK', data: user });
+  } catch (error) {
+    console.error('Error al procesar la solicitud:', error);
+    res.status(500).json({ status: 'Error', message: error });
```

Comment:
If it is an internal, do not return the error message, instead return nothing or 'Internal Server Error'